src/test/docker/centos8/Dockerfile

```diff
@@ -43,6 +43,7 @@ RUN yum -y update \
 	python3-six \
 	python3-yaml \
 	python3-jsonschema \
+	python3-sphinx \
```

Comment:
Does this also need `python3-sphinx_rtd_theme`?  Surprisingly the CI seems to pass without it.

Reply:
Nevermind.  Looks like that is only necessary for `make html` and we only run `make man` in `Makefile.am`.